native/src/main/java/io/ballerina/stdlib/http/transport/contractimpl/sender/states/http2/RequestCompleted.java

```diff
@@ -83,7 +83,9 @@ public void readInboundResponseHeaders(ChannelHandlerContext ctx, Http2HeadersFr
     public void readInboundResponseBody(ChannelHandlerContext ctx, Http2DataFrame http2DataFrame,
                                         OutboundMsgHolder outboundMsgHolder, boolean serverPush,
                                         Http2MessageStateContext http2MessageStateContext) {
-        LOG.warn("readInboundResponseBody is not a dependant action of this state");
+        http2MessageStateContext.setSenderState(new ReceivingEntityBody(http2TargetHandler, http2RequestWriter));
```

Comment:
Shall we add a comment saying, following is needed for the bidirectional streaming?

Reply:
Added in https://github.com/ballerina-platform/module-ballerina-http/pull/796/commits/ce5400904b192a4bff75ecbe68f5d1402e8539da